happiness-backend/api/models/schema.py

```diff
@@ -223,3 +225,22 @@ class ReadsSchema(ma.Schema):
     happiness_id = ma.Int(dump_only=True, required=True)
     user_id = ma.Int(dump_only=True, required=True)
     timestamp = ma.Str(dump_only=True, required=True)
+
+
+class HappinessRangeSchema(ma.Schema):
+    id = ma.Int()
+    page = ma.Int()
+    count = ma.Int()
+    low = ma.Int()
+    high = ma.Int()
+
+
+class HappinessMultiFilterSchema(ma.Schema):
+    user_id = ma.Int()
+    page = ma.Int()
+    count = ma.Int()
+    low = ma.Int(missing=11)
+    high = ma.Int(missing=11)
```

Comment:
why 11 😠

Reply:
can i change it to None in the schema for the optional argument?
heres what the schema looks like at the moment, should i just make all the optional arguments none, like including the text one which is currently the empty string? i assume that would be fine to do right?
  user_id = ma.Int()
    page = ma.Int()
    count = ma.Int()
    low = ma.Int(missing=11)
    high = ma.Int(missing=11)
    start = ma.Str(missing=None)
    end = ma.Str(missing=None)
    text = ma.Str(missing="")